### PR TITLE
[Subgraph] New schema implementation

### LINKED ISSUE #721: Subgraph - Implement new schema design
**Description**
We need to re-design subgraph schema, as well as handlers.

https://app.diagrams.net/#G1WCSJhsWy4irN0wgxlPbePcT0uN5pbtTx

**Motivation**
We have new version of Job Launcher, and Escrow Dashboard design, and this needs changes in subgraph schema, so that we can pull data from.

### PR DESCRIPTION
**Pull request by leric7**
## Description

Subgraph schemas are re-defined based on new design of Job Launcher, and Escrow Dashboard.
https://app.diagrams.net/#G1WCSJhsWy4irN0wgxlPbePcT0uN5pbtTx

## Summary of changes

- Updated subgraph schema definition
- Updated subgraph handlers to properly save data
- Updated Node SDK to properly handle changed subgraph schema
- Updated test cases

## Related issues
[Keywords for linking issues](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests)

Closes #721 

## Operational checklist

- [x] All new functionality is covered by tests
- [x] Any related documentation has been changed or added

### CODE REVIEW

**Review by portuu3 — CHANGES_REQUESTED** [review]
can we create a new subgraph and deploy it? I would suggest to use mumbai

**Review by portuu3 — CHANGES_REQUESTED** [review]

File: packages/sdk/python/human-protocol-sdk/human_protocol_sdk/escrow.py [hunk 1]
```diff
@@ -90,8 +94,7 @@ class EscrowFilter:
 
     def __init__(
         self,
-        address: Optional[str] = None,
-        addressRole: Optional[Role] = None,
+        launcher_address: Optional[str] = None,
```
[the remark this answers is not in the document]
Comment (leric7) [reply packages/sdk/python/human-protocol-sdk/human_protocol_sdk/escrow.py:97]: Will revisit this on #758

File: packages/sdk/typescript/human-protocol-sdk/src/graphql/queries/escrow.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    escrows(where: { launcher: $launcherAddress }) {
+      ...EscrowFields
+    }
+  }
+  ${ESCROW_FRAGMENT}
+`;
+
+export const GET_FILTERED_ESCROWS_QUERY = gql`
+  query GetFilteredEscrows(
+    $launcherAddress: String
```
[the remark this answers is not in the document]
Comment (leric7) [reply packages/sdk/typescript/human-protocol-sdk/src/graphql/queries/escrow.ts:37]: Will revisit this on #758

File: packages/sdk/typescript/human-protocol-sdk/test/escrow.test.ts [hunk 3]
```diff
@@ -1416,7 +1416,7 @@ describe('EscrowClient', () => {
       const requesterAddress = FAKE_ADDRESS;
 
       await expect(
-        escrowClient.getEscrowsFiltered({ address: requesterAddress })
+        escrowClient.getEscrowsFiltered({ launcherAddress: requesterAddress })
```
[the remark this answers is not in the document]
Comment (leric7) [reply packages/sdk/typescript/human-protocol-sdk/test/escrow.test.ts:1419]: Will revisit this on #758

File: packages/sdk/typescript/human-protocol-sdk/test/escrow.test.ts [hunk 4]
```diff
@@ -1429,7 +1429,7 @@ describe('EscrowClient', () => {
       );
 
       const results = await escrowClient.getEscrowsFiltered({
-        address: requesterAddress,
+        launcherAddress: requesterAddress,
```
[the remark this answers is not in the document]
Comment (leric7) [reply packages/sdk/typescript/human-protocol-sdk/test/escrow.test.ts:1432]: Will revisit this on #758

File: packages/sdk/typescript/human-protocol-sdk/src/interfaces.ts [hunk 5]
```diff
@@ -24,7 +24,7 @@ export interface IStaker {
 }
 
 export interface IEscrowsFilter {
-  address?: string;
+  launcherAddress?: string;
```
[the remark this answers is not in the document]
Comment (leric7) [reply packages/sdk/typescript/human-protocol-sdk/src/interfaces.ts:27]: Will revisit this on #758

**Review by portuu3 — APPROVED** [review]

**Pull request merged**